summarize/process_summaries.py

```diff
@@ -0,0 +1,382 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# ***************************************************************************80
+#
+# python ./summarize/process_summaries.py
```

Comment:
process summaries
summarize transcripts
summarize long transcripts

all very similar names - can you expand the flowerbox to explain what each does and how they are related?